assembl/graphql/schema.py

```diff
@@ -115,7 +115,6 @@ def resolve_root_idea(self, args, context, info):
         return root_thematic
 
     def resolve_vote_session(self, args, context, info):
-
         discussion_phase_id = args.get('discussion_phase_id')
         discussion_phase = models.DiscussionPhase.get(discussion_phase_id)
         # TODO: see if we can avoid this next(iter( thing with a one-to-one relationship
```

Comment:
About this TODO, you may simply want uselist=False on the relationship, no?

Reply:
Perfect, thank you !